packages/big-design/src/components/Box/Box.tsx

```diff
@@ -25,8 +25,6 @@ interface PrivateProps {
 
 const RawBox: React.FC<BoxProps & PrivateProps> = (props) => <StyledBox ref={props.forwardedRef} {...props} />;
 
-export const Box = memo(
-  forwardRef<HTMLDivElement, BoxProps>((props, ref) => <RawBox {...props} forwardedRef={ref} />),
-);
+export const Box = memo(forwardRef<HTMLDivElement, BoxProps>((props, ref) => <RawBox {...props} forwardedRef={ref} />));
```

Comment:
The updated version of `prettier` cleans up some of these "add newline" errors we had in the past.